### DISCUSSION TITLE
disable syncs when hosted locally

### DISCUSSION THREAD

**Discussion by davew94** [hoppscotch/hoppscotch#2822] [Q&A]
Hi,

In my organisation we only want members of staff to sync with hoppscotch using one method (microsoft). Is it possible to disable syncs (github/google) if hoppscoth is hosted within our company network?

**Comment by AndrewBastin** [comment]
Well, it is not possible currently, but we are planning to introduce a self hosted version that can be widely hosted in companies, we will try to include this feature in that.

@hoppscotch/backend pinging to just discuss this later on internally.

**Reply by arturssmirnovs** [reply]
@AndrewBastin Awesome news for self hosted version, is there any ETA? :)

**Reply by AndrewBastin** [reply]
We can't give anything fully concrete yet but hopefully early 2023. We are also reaching out to companies who want to adopt Hoppscotch for their thoughts and feature requests, if you are interested, please reach out to me or Liyas over on our Discord or Telegram server.

**Comment by dancabper** [comment]
Hi, I try to login with the Microsoft credentials but I have put the Microsoft credentials and when I login it returns the following error "AADSTS900971: No reply address provided". ¿What is the correct reply address url?
